packages/vue/README.md

```diff
@@ -100,3 +104,126 @@ type useSharedState = <
+
+Create a file called `index.ts|js` in the `src` directory where your composable app is. For example:
+Create a file called `index.ts|js` in the `src` directory where your composable app is. For example:
+
+```
+my-monorepo/
+├- apps/
+│  ├─ vue-host/
+├─ composable-apps/
+│  ├─ vue-app-1/
+│  │  ├─ package.json
+│  │  ├─ src/
+│  │  │  ├─ VueApp1.vue
+│  │  │  ├─ index.ts 👈
+├─ package.json
+```
+
+:::tip
+
+Read the recommended setup in our [getting started page](../../docs/getting-started/#recommended-setup) if you want to create a similar monorepo structure
+
+:::
+
+Call `createApp` with the root component of your app, for example:
+
+```ts
+// my-monorepo/composable-apps/vue-app-1/src/index.ts
+import { createApp } from "@leanjs/vue-router";
+
+import RemoteVue1 from "./RemoteVue1.vue";
+
+export default createApp(RemoteVue1, { appName: "RemoteVue1" });
+```
+
+Hello world example of the `ReactApp1` imported above
+
+```tsx
+<template>
+  <div>
+    <h1>Hello Vue composable-app 1</h1>
+  </div>
+</template>
+```
+
+Create a file called `selfHosted.ts|js` in the `src` directory where your composable app is, for example:
+
+```
+my-monorepo/
+├- apps/
+│  ├─ vue-host/
+├─ composable-apps/
+│  ├─ vue-app-1/
+│  │  ├─ package.json
+│  │  ├─ src/
+│  │  │  ├─ VueApp1.vue
+│  │  │  ├─ index.ts 👈
+│  │  │  ├─ selfHosted.ts 👈
+├─ package.json
+```
+
+Export a `createRuntime` function from the `selfHosted.ts|js` file. This is the runtime that will be used when the app runs in isolation, meaning without a host.
+
+```ts
+// my-monorepo/composable-apps/vue-app-1/src/selfHosted.ts
+export { createRuntime } from "@my-org/runtime-vue";
+```
+
+:::info
+
+Read [@leanjs/core](../core/README.md#basic-usage) if you have not already created your own `createRuntime` function
+
+:::
+
+## Components
+
+### `HostProvider`
+
+You have to **call [createRuntimeBindings](#createruntimebindings) to create a `HostProvider` component** before you use it. `HostProvider` stores in a React context values that are shared across apps hosted in the same component tree. Props:
+
+#### `runtime` prop - required
+
+Your Lean [runtime](../core/README.md).
+
+#### `origin` prop - optional
+
+[Origin](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin) where your remote composable apps are. During development, use the address where you run your Lean [proxy dev server](../cli/README.md#proxy-dev-server). Use the address of your CDN in production, e.g. `https://cdn.example.com`.
+
+Example
+
+```tsx
+<template>
+    <HostProvider :runtime="runtime" :origin="origin">
+      <NuxtPage />
+    </HostProvider>
+</template>
+
+<script setup lang="ts">
+  import { createRuntime, HostProvider } from "./shared-runtime";
+
+  const runtime = createRuntime({
+    context: { appName: "AppExample" },
+  });
+
+  const origin = process.env.MICROFRONTENDS_ORIGIN;
+</script>
+```
+
+### `Host`
+
+It hosts a composable app in a React host.
```

Comment:
It hosts a composable app in a **Vue** host.

Reply:
done